lib/tasks/protractor.rake

```diff
@@ -118,7 +140,7 @@ namespace :db do
   namespace :test do
     desc "seed only the test database (Task provided by protractor-rails)"
     task :seed do
-      system "rake db:seed RAILS_ENV=test"
+      system "rake db:seed_fu RAILS_ENV=e2e_test"
```

Comment:
e2e_test is also a non-standard rails env. Can you motivate why you have elected to do this?